packages/jellyfish-api-core/__tests__/category/icxorderbook/closeOffer.test.ts

```diff
@@ -175,15 +175,10 @@ describe('ICXOrderBook.closeOffer', () => {
     expect((ordersAfterMakeOffer as Record<string, ICXOfferInfo>)[makeOfferTxId].status).toStrictEqual(ICXOrderStatus.OPEN)
 
     // input utxos
-    const utxos = await container.call('listunspent', [1, 9999999, [accountBTC], true])
-    const inputUTXOs: UTXO[] = utxos.map((utxo: UTXO) => {
-      return {
-        txid: utxo.txid,
-        vout: utxo.vout
-      }
-    })
+    const inputUTXOs = await container.fundAddress(accountBTC, 10)
```

Comment:
```suggestion
    const inputUTXO = await container.fundAddress(accountBTC, 10)
```

there is no s to this since it's just one

Reply:
FYI, don't need resolve now